packages/api/src/services/emp-state.ts

```diff
@@ -140,6 +140,8 @@ export default (config: Config, appState: Dependencies) => {
         staticState = await readEmpStaticState(instance, address);
         // create active emp with static/dynamic state
         await emps.active.create({ ...staticState, ...dynamicState });
+      } else {
+        await emps.active.update(address, dynamicState);
```

Comment:
emps suffered from the same issue, the case where they existed and were active, they were never being re-updated with dynamic data